setup.py

```diff
@@ -26,10 +26,11 @@
         "Operating System :: OS Independent",
         "Development Status :: 1 - Planning",
     ],
+    py_modules=["_icepyx_version"],
     use_scm_version={
         "fallback_version": "unknown",
-        "local_scheme": "node-and-date",
-        "write_to": "_version.py",
+        "local_scheme": "no-local-version",
```

Comment:
```suggestion
        "local_scheme": "node-and-date",
```
Need to keep this as node-and-date because the publish to pypi Github Action workflow expects it.